src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-card/info-card.component.scss

```diff
@@ -21,9 +21,12 @@ $card-font-max-size: 21px;
     padding-top: 40px !important;
 
     .card-title {
-      left: 0;
+      border-bottom: 1px solid bd.$white;
```

Comment:
```suggestion
```

Reply:
@tspmelo ```border-bottom: 1px solid bd.$white;``` is intended to activate the underline by giving the desired color in downstream, but it can be removed.